fifo/producer.go

```diff
@@ -35,77 +38,169 @@ func NewProducer(config *kafka.ConfigMap, reg *srclient.SchemaRegistryClient, to
 	}, nil
 }
 
-func (p *Producer) Run(reader zio.Reader) error {
+func (p *Producer) HeadOffset() (kafka.Offset, error) {
+	_, high, err := p.producer.QueryWatermarkOffsets(p.topic, 0, 1000*10)
+	return kafka.Offset(high), err
+}
+
+func (p *Producer) Run(ctx context.Context, reader zio.Reader) error {
+	fmt.Printf("producing messages to topic %q...\n", p.topic)
```

Comment:
Should a library package be printing to stdout?

Reply:
Right now, the entirety of the zinger package is a CLI tool so I vote to fix this later.

Reply:
I made issue #21